remoteappmanager/file_config.py

```diff
@@ -57,6 +64,34 @@ class FileConfig(HasTraits):
         default_value=paths.static_dir,
         help="The path where to search for static files")
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Sets the default of the docker configuration options from the
+        # current environment. These will possibly be overridded by
+        # the appropriate entries in the configuration file when parse_file
+        # is invoked
+
+        env = os.environ
+
+        self.docker_host = env.get("DOCKER_HOST", "")
+        if self.docker_host == "":
+            self.docker_host = "unix://var/run/docker.sock"
+
+        self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
+
+        cert_path = env.get("DOCKER_CERT_PATH", "")
+        if self.tls_verify or cert_path != "":
+            if cert_path == "":
+                cert_path = os.path.join(os.path.expanduser("~"), ".docker")
+
+            self.docker_host = self.docker_host.replace('tcp://', 'https://')
```

Comment:
That's how docker does it...

Which brings me to the point if we should replace also the config file entry if wrong, or should we just go straight as it says....

At the moment, what's in the file is used verbatim.